src/pages/ReimbursementAccount/BankInfo/substeps/Plaid.js

```diff
@@ -0,0 +1,124 @@
+    reimbursementAccountDraft: {},
+    plaidData: PlaidDataProps.plaidDataDefaultProps,
+};
+
+function Plaid({reimbursementAccount, reimbursementAccountDraft, onNext, plaidData}) {
+    const {translate} = useLocalize();
+    const isFocused = useIsFocused();
+
+    const validate = useCallback((values) => {
+        const errorFields = {};
+        if (!values.acceptTerms) {
+            errorFields.acceptTerms = 'common.error.acceptTerms';
+        }
+
+        return errorFields;
+    }, []);
+
+    useEffect(() => {
+        const plaidBankAccounts = lodashGet(plaidData, 'bankAccounts') || [];
+        if (isFocused || plaidBankAccounts.length) {
+            return;
+        }
+        BankAccounts.setBankAccountSubStep(null);
+    }, [isFocused, plaidData]);
+
+    const handleNextPress = useCallback(() => {
+        const selectedPlaidBankAccount = _.findWhere(lodashGet(plaidData, 'bankAccounts', []), {
+            plaidAccountID: lodashGet(reimbursementAccountDraft, 'plaidAccountID', ''),
+        });
+
+        const bankAccountData = {
+            [bankInfoStepKeys.ROUTING_NUMBER]: selectedPlaidBankAccount.routingNumber,
+            [bankInfoStepKeys.ACCOUNT_NUMBER]: selectedPlaidBankAccount.accountNumber,
+            [bankInfoStepKeys.PLAID_MASK]: selectedPlaidBankAccount.mask,
+            [bankInfoStepKeys.IS_SAVINGS]: selectedPlaidBankAccount.isSavings,
+            [bankInfoStepKeys.BANK_NAME]: lodashGet(plaidData, 'bankName') || '',
+            [bankInfoStepKeys.PLAID_ACCOUNT_ID]: selectedPlaidBankAccount.plaidAccountID,
+            [bankInfoStepKeys.PLAID_ACCESS_TOKEN]: lodashGet(plaidData, 'plaidAccessToken') || '',
+        };
+
+        ReimbursementAccount.updateReimbursementAccountDraft(bankAccountData);
+        onNext();
+    }, [plaidData, reimbursementAccountDraft, onNext]);
+
+    const bankAccountID = getDefaultValueForReimbursementAccountField(reimbursementAccount, bankInfoStepKeys.BANK_ACCOUNT_ID) || 0;
+    const selectedPlaidAccountID = lodashGet(reimbursementAccountDraft, 'plaidAccountID', '');
+
+    return (
+        <Form
+            formID={ONYXKEYS.REIMBURSEMENT_ACCOUNT}
+            validate={validate}
+            onSubmit={handleNextPress}
+            scrollContextEnabled
+            submitButtonText={translate('common.next')}
+            style={[styles.mh5, styles.flexGrow1]}
+            isSubmitButtonVisible={Boolean(selectedPlaidAccountID) && !_.isEmpty(lodashGet(plaidData, 'bankAccounts'))}
+        >
+            <AddPlaidBankAccount
+                onSelect={(plaidAccountID) => {
+                    ReimbursementAccount.updateReimbursementAccountDraft({plaidAccountID});
+                }}
+                plaidData={plaidData}
+                onExitPlaid={() => BankAccounts.setBankAccountSubStep(null)}
+                allowDebit
+                bankAccountID={bankAccountID}
+                selectedPlaidAccountID={selectedPlaidAccountID}
+                isDisplayedInNewVBBA
```

Comment:
will this come from somewhere else, or is it just a placeholder? the default value is set to `false`

Reply:
Everything is in place. This component is used somewhere esle in the codebase as well. This is the only place where we want to display updated UI inside of it thus prop set to true